Automattic/jetpack
# Boost: Add cache clear actions

Fixes #35784

## Proposed changes:

* Added actions that third party code can trigger to clear cache.

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
None

## Testing instructions:






Test by using the following to see if cache get's cleared

```php
do_action( 'jetpack_boost_clear_page_cache_all' ); // delete entire cache
do_action( 'jetpack_boost_clear_page_cache_url', $url ); // delete cache for URL, including paged archive pages
do_action( 'jetpack_boost_clear_page_cache_post', $post_id ); // delete cache for a post, including comment archives.
do_action( 'jetpack_boost_clear_page_cache_home' ); // delete cache files for homepage and paged archives.
```
